images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -270,3 +270,8 @@ function Get-SwigVersion {
     $swigVersion = $Matches.Version
     return "Swig $swigVersion"
 }
+
+function Get-YqVersion {
```

Comment:
that should be in the linux folder, not windows :)

Reply:
Oh dammit, you're right :)
Fixed now - hopefully!